nipoppy_cli/nipoppy/data/examples/sample_global_config-all_pipelines.json

```diff
@@ -89,16 +96,17 @@
             },
             "STEPS": [
                 {
-                    "INVOCATION_FILE": "[[NIPOPPY_DPATH_INVOCATIONS]]/fmriprep-23.1.3.json"
+                    "INVOCATION_FILE": "[[NIPOPPY_DPATH_INVOCATIONS]]/[[NIPOPPY_PIPELINE_NAME]]-[[NIPOPPY_PIPELINE_VERSION]].json",
+                    "DESCRIPTOR_FILE": "[[NIPOPPY_DPATH_DESCRIPTORS]]/[[NIPOPPY_PIPELINE_NAME]]-[[NIPOPPY_PIPELINE_VERSION]].json"
                 }
             ]
         },
         {
             "NAME": "fmriprep",
             "VERSION": "20.2.7",
             "CONTAINER_INFO": {
-                "PATH": "[[NIPOPPY_DPATH_CONTAINERS]]/fmriprep_20.2.7.sif",
-                "URI": "docker://nipreps/fmriprep:20.2.7"
+                "FILE": "[[NIPOPPY_DPATH_CONTAINERS]]/[[NIPOPPY_PIPELINE_NAME]]_[[NIPOPPY_PIPELINE_VERSION]].sif",
+                "URI": "docker://nipreps/[[NIPOPPY_PIPELINE_NAME]]:[[NIPOPPY_PIPELINE_VERSION]]"
             },
             "CONTAINER_CONFIG": {
                 "ENV_VARS": {
```

Comment:
So by default we have this Templateflow directory for fMRIPrep. For Freesurfer license it doesn't strictly have to be an env var since fMRIPrep has the `--fs-license-file` argument. It doesn't really make a difference (especially with the replacement things we are doing) so if you prefer having Freesurfer license file also as an env var instead of directly passed to fMRIPrep we can do that

Reply:
No it's fine. I was just checking if there was a way to move this somewhere else if it's going to be empty by default, or offer some useful example for people seeing it for the first time.

Reply:
It's not empty though? For fMRIPrep we have the `TEMPLATEFLOW_HOME` env var. Maybe this is just being cut off by GitHub because those lines haven't been changed in this PR

Reply:
Yes, you are right. It cuts off exactly the `TEMPLATEFLOW_HOME` env var. The lines around it were still there despite not being changed in the PR.. so got confused. Should be good!